testgrid/config.yaml

```diff
@@ -4054,42 +4054,42 @@ dashboards:
     test_group_name: ci-kubernetes-e2e-gke-canary
   - name: gke-cos-containerd-alpha-cluster
     test_group_name: ci-kubernetes-e2e-cos-containerd-gke-alpha-cluster
-  # K8S 1.8 branch on GKE
-  - name: gci-gke-ingress-1.8
+  # K8S 1.9 branch on GKE
```

Comment:
(it won't update comments.... hummm... let's see)

Reply:
The comments are pretty useless. I updated them to use stableN/beta, etc.